code/datums/mind.dm

```diff
@@ -236,10 +230,18 @@
 	remove_nukeop()
 	remove_wizard()
 	remove_cultist()
-	remove_rev()
+	remove_shadowling()

```

Comment:
shame this isn't a proc on the universe.

Reply:
kek